site/README.md

```diff
@@ -151,6 +162,20 @@ Retrieves the user object of the currently logged in user.
 - Request Response:
   - `user`: the user currently authenticated with the API
 
+### `GET /api/me/apiKeys` [authenticated]
+
+Retrieves metadata on the API keys associated with the authenticated user.
+
+- Request Response:
+  - `apiKeys`: metadata on API keys (the key itself is only visible at the point of generation)
+
+### `POST /api/me/generateApiKey` [authenticated]
+
+Generates a new API key for the authenticated user, and revokes any others.
+
```

Comment:
I think this is missing the definition of the `Request Body`, which requires a `displayName` in the request body

Reply:
Thanks, added